resources.qrc

```diff
@@ -9,5 +9,6 @@
         <file>qml/2048.js</file>
         <file>qml/Tile.qml</file>
         <file>res/fonts/DroidSansFallback.ttf</file>
+        <file>ts/2048-qt_pt_BR.qm</file>
```

Comment:
wrongly placed, whould be between `<file>ts/2048-qt_pl_PL.qm</file>` and ` <file>ts/2048-qt_ru_RU.qm</file>` -> line 6